### PR TITLE
Cursorless scope side bar

### PR DESCRIPTION
**Pull request by pokey**
- Depends on https://github.com/cursorless-dev/cursorless/pull/1941
- Depends on #1946

## Checklist

- [-] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/) (see comment below)
- [x] Add descriptions to section headings (eg present, not present, etc)
- [x] File issue about using lang icons for lang-specific scopes
- [x] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [x] I have not broken the cheatsheet
- [x] File issue for showing scopes touching, as well as equal to, selection
- [x] Checkboxes from original #1663

### CODE REVIEW

File: packages/common/src/cursorlessCommandIds.ts [hunk 1]
```diff
@@ -69,6 +69,12 @@ export const cursorlessCommandDescriptions: Record<
     "Resume test case recording",
   ),
   ["cursorless.showDocumentation"]: new VisibleCommand("Show documentation"),
+  ["cursorless.showScopeVisualizer"]: new VisibleCommand(
+    "Show the scope visualizer",
+  ),
```
Comment (pokey) [inline packages/common/src/cursorlessCommandIds.ts:74]: File issue to make this work with no args

Comment (pokey) [reply packages/common/src/cursorlessCommandIds.ts:74]: https://github.com/cursorless-dev/cursorless/issues/1978

**Comment by pokey** [comment]
Ok this one is also good to go @AndreasArvidsson; will need to merge in behind a couple others tho so please don't merge

**Comment by pokey** [comment]
I didn't add tests for this one, because most of the hard logic is tested in #1941. This one is trying hard to be pure presentation, and unfortunately testing it is quite hard until we have a setup where we delay initialisation of the extension to allow test harness to be put in place

**Review by AndreasArvidsson — APPROVED** [review]

**Pull request merged**